src/CoreWCF.NetTcp/src/CoreWCF/Channels/Framing/FramingDecoder.cs

```diff
@@ -386,6 +386,14 @@ public static string GetString(FramingEncodingType type)
                     return FramingEncodingString.Binary;
                 case FramingEncodingType.BinarySession:
                     return FramingEncodingString.BinarySession;
+                //case FramingEncodingType.ExtendedBinaryGZip:
+                //    return FramingEncodingString.ExtendedBinaryGZip;
```

Comment:
I believe these commented out variations are needed to support compression when using TransferMode Streamed as that doesn't maintain a session between requests. So I think you need to include the non-session variants too.

Reply:
Thanks for pointing out TransferMode.Streamed. I've uncommented those variations and added appropriate test cases.